Tests/scripts/validate_premium_packs.py

```diff
@@ -0,0 +1,239 @@
+"""Validate premium packs.
+
+Check index.json file inside the index.zip archive in the cloud.
+Validate no missing ids are found and that all packs have a positive price.
+Validate commit hash is in master's history.
+Check the server configured on master.
+Validate the pack id's in the index file are present on the server and the prices match.
+"""
+import demisto_client
+import argparse
+import logging
+import ast
+import sys
+
+from Tests.scripts.validate_index import log_message_if_statement, get_index_json_data
+from Tests.configure_and_test_integration_instances import Build, Server
+from Tests.Marketplace.marketplace_services import GCPConfig
+from Tests.scripts.utils.log_util import install_logging
+from Tests.test_content import get_json_file
+from pprint import pformat
+
+DEFAULT_PAGE_SIZE = 50
+
+
+def options_handler():
+    parser = argparse.ArgumentParser(description='Test for validating premium packs on servers.')
+    parser.add_argument('--ami_env', help='The AMI environment for the current run. Options are '
+                                          '"Demisto 6.0", "Demisto Marketplace". The server url is determined by the'
+                                          ' AMI environment.', default="Demisto Marketplace")
+    parser.add_argument('-e', '--extract_path',
+                        help=f"Full path of folder to extract the {GCPConfig.INDEX_NAME}.zip to",
+                        required=True)
+    parser.add_argument('-pb', '--production_bucket_name', help="Production bucket name", required=True)
+    parser.add_argument('-sa', '--service_account', help="Path to gcloud service account", required=True)
+    parser.add_argument('-s', '--secret', help='Path to secret conf file', required=True)
+
+    options = parser.parse_args()
+    return options
+
+
+def get_paid_packs_page(client: demisto_client,
+                        page: int = 0,
+                        size: int = DEFAULT_PAGE_SIZE,
+                        request_timeout: int = 999999):
+    """Get premium packs from client.
+
+    Trigger an API request to demisto server.
+    Request is identical to checking the premium box through the marketplace GUI.
+
+    Args:
+        client: The demisto client to the preform request on.
+        page: Page number of the request.
+        size: Number of packs to bring each time.
+        request_timeout: Timeout of API request
+
+    Returns:
+        (Dict: premium packs as found in the server, int: Total premium packs that exist)
+        (None, 0) if no premium packs were found.
+    """
+    request_data = \
+        {
```

Comment:
no need for the over indent and hanged line here, should be:
```python
request_data = {
    ...
}
```

Reply:
This was for a demo run, not really ready for review. You can review now though :)